testing-utils/src/main/java/no/sikt/nva/data/report/testing/utils/generator/TestData.java

```diff
@@ -260,6 +260,12 @@ private static Model getModelWithAppliedView(TestNviCandidate candidate) {
         return new ViewCompiler(model).extractView(id);
     }
 
+    private void sortByPublicationId() {
+        if (publicationTestData.size() > ONE) {
```

Comment:
You're checking something that it is not necessary to check

Reply:
Good point, i was getting `UnsupportedOperationException` on sorting because I implemented `publicationTestData` and `nviTestData` as an immutable `List` in the new constructor. Changed to `ArrayList` now in latest commit.